tvtk/wrapper_gen.py

```diff
@@ -284,6 +285,8 @@ def _write_prelims(self, node, out):
         from tvtk.array_handler import deref_array
         from tvtk.tvtk_classes.tvtk_helper import wrap_vtk
 
+        nan = float('nan')
```

Comment:
Some arrays had `nan` in them, and without this in the preamble, I got unknown variable issues (probably/possibly coming from the `*extra_values` where there were `nan` in the list)